web/forms.py

```
@@ -527,6 +527,50 @@ def clean(self):
+    
+    expected_output = forms.CharField(
+        widget=TailwindTextarea(attrs={'rows': 3, 'placeholder': 'Expected output for coding questions'}),
+        required=False,
+        help_text="Expected output for coding questions"
+    )
+    
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields['question_type'].widget.attrs.update({'class': 'form-control'})
+        
+    def clean(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `clean`

(ANN202)